### ISSUE TITLE
Neat omega mixin for tablet resolution  variable required

### ISSUE THREAD

**Issue by alecky** [AusDTO/gov-au-ui-kit#194]
## When nesting omegas between breakpoints the min-with and max-width needs to be supplied

As per the article on [here](https://github.com/thoughtbot/neat/issues/116) which states that the min and max width needs to be provided. We need a mobile only and tablet only variable for these cases.

For example: 

``````
@include media(min-width 768px max-width (1200px - 1)) {
      @include span-columns(6 of 12);
      @include omega(2n);
}
@include media($desktop) {
      @include span-columns(4);
      @include omega(4n);
}```
``````

**Comment by TrebBrennan** [comment]
Nice find @alecky this is something we've been trying to resolve.

We're likely to try something like this:
https://github.com/thoughtbot/neat#how-do-i-use-omega-in-a-mobile-first-workflow

**Comment by TrebBrennan** [comment]
Possible fix in PR#210
https://github.com/AusDTO/gov-au-ui-kit/pull/210/files#diff-e529945c9defd9ee06cd758ccabf8f3fL83

**Comment by TrebBrennan** [comment]
Should be resolved in v1.5.0
https://github.com/AusDTO/gov-au-ui-kit/pull/210/files#diff-e529945c9defd9ee06cd758ccabf8f3fL51
